examples/particle/main.rs

```diff
@@ -183,7 +201,7 @@ impl Example {
             .update_textures(&mut self.command_encoder, gui_textures, &self.context);
         self.particle_system.update(&mut self.command_encoder);
 
-        if self.sample_count <= 1 {
+        if self.sample_count <= 1 && !self.export_image {
```

Comment:
I don't understand this part. Are we going the MSAA route whenever `self.export_image` is true?

Reply:
Yes since the MSAA image is configured to export (if enabled). So it would have to render to that for it to be exported even if its only sampled once.